### PR TITLE
fix(TripPlanner): Remove the header from the accordion on the trip pl…

### PR DESCRIPTION
**Pull request by kotva006**
…anner

#### Summary of changes
**Asana Ticket:** [Repeated heading 3 label for itinerary and content](https://app.asana.com/0/555089885850811/1202300116369146)

The `aria-role=header` attribute has been set as optional for the `Accordion` component.
This is to stop the screen read from reading each part of the trip with header title

Note: The role is optional, but it needs to be deliberately opted out of, as it defaults to setting the header given no information

---

Before getting review, please check the following:

* [ ] Does frontend functionality render and work correctly in IE?
* [ ] Have we load-tested any new pages or internal API endpoints that will receive significant traffic? [See load testing docs](./../apps/site/load_tests/README.md)
* [ ] Are interactive elements accessible to screen readers?
* [ ] Have you checked for tech debt you can address in the area you're working in?
* [ ] If this change involves routes, does it work correctly with pertinent "unusual" routes such as the combined Green Line, Silver Line, Foxboro commuter rail, and single-direction bus routes like the 170?
* [ ] Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?

### CODE REVIEW

**Review by i0sea — APPROVED** [review]

**Comment by lund5000** [comment]
Testing with NVDA, this does prevent the accordion from being read when browsing through headings (as Ryan feared it might). That seems undesirable. 

Is it possible and advisable to set `heading="role"` and `aria-level="3"` on the `div.c-accordion-ui__heading` inside/instead of `div.panel`? Based on manual editing in dev tools, it seems to produce a more desirable result: the accordion expand/collapse button is still read as heading level 3 but the steps within are not.

File: apps/site/assets/js/app.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ if (window.sentry) {
     dsn: window.sentry.dsn,
     environment: window.sentry.environment,
     autoSessionTracking: false,
-    sampleRate: 0.10, // error sampling - might increase later
+    sampleRate: 0.1, // error sampling - might increase later
```
Comment (kotva006) [inline apps/site/assets/js/app.js:56]: Does this need to be `0.10`?

File: apps/site/assets/ts/components/Accordion.tsx [hunk 2]
```diff
@@ -19,15 +19,13 @@ export const AccordionNoJS = ({
   children: ReactElement<HTMLElement>;
 }): ReactElement<HTMLElement> => (
   <div className="c-accordion-ui">
-    <div role="heading" aria-level={3}>
```
Comment (kotva006) [inline apps/site/assets/ts/components/Accordion.tsx:22]: I removed this div, because there doesn't appear to be a heading/title in the NoJS accordion

**Review by thecristen — APPROVED** [review]
:sparkles: I think this will do the trick! Thanks for this revision.

**Pull request merged**